lib/src/vector_tile_provider.dart

```diff
@@ -29,8 +30,7 @@ class NetworkVectorTileProvider extends VectorTileProvider {
   ///  confused with the maximum zoom of the map widget. The map widget will
   ///  automatically use vector tiles from lower zoom levels once the maximum
   ///  supported by this provider is reached.
-  NetworkVectorTileProvider(
-      {required String urlTemplate, this.httpHeaders, int maximumZoom = 16})
+  NetworkVectorTileProvider({required String urlTemplate, this.httpHeaders, int maximumZoom = 16})
```

Comment:
There are a lot of changes that appear to be just formatting, which makes it a lot harder to see what has changed. Can you remove all formatting changes so that it's easier to review?

Reply:
Yeah, that was my mistake. I forgot to turn off my auto formatter 😅